src/Native/Runtime/inc/varint.h

```diff
@@ -1,6 +1,11 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
+
+#ifdef _WASM_
+#include "emscripten.h"
```

Comment:
I wonder if we should just define a `GET_UNALIGNED_VAL32` macro like we have in CoreCLR that expands to the right thing on WASM instead of adding WASM specific stuff here. I don't think this is the only place that will need to do unaligned reads. @jkotas what do you think?

Reply:
Agree that it would be nice to define a macro for this.